src/main/xar-resources/data/webdav/webdav.xml

```diff
@@ -1,244 +1,301 @@
+            <note>
+                <para>Note on SSL connections in oXygen: You may encounter an error: <emphasis>"Error:
+                    sun.security.validator.ValidatorException: PKIX path building failed:
+                    sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid
+                    certification path to requested target."</emphasis>
+                </para>
+                <para>In order to avoid this error, you'll need to add the Server certificate to your
+                    trusted certificates keystore (<link xlink:href="http://java.sun.com/j2se/1.5.0/docs/guide/security/jsse/JSSERefGuide.html#CustomizingStores" condition="_blank">more info</link>).</para>
+            </note>
+            <warning>
+                <para>When saving a document to the WebDAV server via oXygen, remember to double check that
+                    the document's path in the File URL field at the top of the dialog does really contain the
+                    URL to which you want to save the document to. It happens too easily that when you click
+                    on another folder or resource by mistake, you overwrite the wrong resource. </para>
+            </warning>
+        </sect2>
+        
+        <!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
+        
+        <sect2>
+            <title>davfs2</title>
+            <para>Recent eXist versions present by default, through WebDAV, the size of XML documents as a multiple of internal pagesize. 
+                This approximated size is not managed correctly by davfs2. You can list XML files (<code>ls -l</code>) but you cannot 
+                view or edit them. All other file types (plain text files, images, etc) are working as expected with davfs2.   
+            </para>
+            <para>The following message is found in davfs2 logs (syslog/journalctl) :              
+                <errortext>Could not read response body: connection was closed by server</errortext>
+            </para>
+            <para>
+                In order to access XML files stored in eXist from
+                a mounted directory in a filesystem thanks to dafvs2, you have to add the following additional Java parameters 
+                when starting eXist : </para>
+            
+            <itemizedlist>
+                <listitem><para>org.exist.webdav.PROPFIND_METHOD_XML_SIZE=EXACT</para></listitem>
+                <listitem><para>org.exist.webdav.GET_METHOD_XML_SIZE=EXACT</para></listitem>
+            </itemizedlist>
+            
+            <para>Please note that if eXist is installed as a service (systemd) the configuration has to be done in <filename>/tools/yajsw/conf/wrapper.conf</filename></para>
+                        
+            <programlisting>                
```

Comment:
Please create a separate `.txt` file for this program listings see [author reference](https://exist-db.org/exist/apps/doc/author-reference#listings)